### PR TITLE
 Migrate ApplyNow Components to shadcn/Tailwind 

### PR DESCRIPTION
**Pull request by 0xgun**
DEPENDS on #14043

## Related Issue

#13946

### CODE REVIEW

File: src/components/Translatathon/ApplyNow.tsx [hunk 1]
```diff
@@ -13,22 +13,22 @@ export const ApplyNow = () => {
   const dateToday = new Date()
   const deadline = new Date(APPLICATION_END_DATE)
 
-  if (dateToday < deadline) {
+  if (dateToday || deadline) {
```
Comment (pettinarip) [inline src/components/Translatathon/ApplyNow.tsx:16]: Why would we want to change this?

File: src/components/Translatathon/ApplyNow.tsx [hunk 2]
```diff
@@ -13,22 +13,22 @@ export const ApplyNow = () => {
...
     return (
-      <Box pt={12}>
+      <div className="pt-13">
```
Comment (pettinarip) [inline src/components/Translatathon/ApplyNow.tsx:18]:
```suggestion
      <div className="pt-12">
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
@0xgun left one question. The rest looks good.

Seems that this PR was branched off #14043. Will mark it as blocked until that one gets merged.

File: src/components/Translatathon/ApplyNow.tsx (hunk 1 above)
Comment (0xgun) [reply src/components/Translatathon/ApplyNow.tsx:16]: I forgot to change it back .

**Comment by 0xgun** [comment]
> @0xgun left one question. The rest looks good.
> 
> Seems that this PR was branched off #14043. Will mark it as blocked until that one gets merged.

@pettinarip pushed the changes

File: src/components/StablecoinsTable.tsx [hunk 3]
```diff
@@ -45,68 +43,58 @@ const StablecoinsTable = ({
   }
 
   return (
-    <Table variant="unstyled" my={8} bg="background.base" mb={8} minW="720px">
-      <Thead bg="background.highlight" color="body.medium">
-        <Tr>
+    <Table className="my-8 mb-8 min-w-[720px] bg-background">
```
Comment (pettinarip) [inline src/components/StablecoinsTable.tsx:46]:
```suggestion
    <Table className="my-8 min-w-[720px] bg-background">
```

File: src/components/Translatathon/ApplyNow.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
-import { Box, Flex } from "@chakra-ui/react"
-
 import { ButtonLink } from "@/components/Buttons"
```
Comment (pettinarip) [inline src/components/Translatathon/ApplyNow.tsx:1]: We need to use the [new ButtonLink component](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/buttons/Button.tsx#L135) in this case.

**Review by pettinarip — CHANGES_REQUESTED** [review]
@0xgun left one last thing.

File: src/components/Translatathon/ApplyNow.tsx (hunk 4 above)
Comment (0xgun) [reply src/components/Translatathon/ApplyNow.tsx]: made required changes

**Review by pettinarip — APPROVED** [review]

**Pull request merged**